index_datas/new_import_data_v2.py

```diff
@@ -0,0 +1,162 @@
+
```

Comment:
**[Question, Required]** I don't understand why this file exsts. It is identical to `import_datas.py` there should one be ONE copy of this code. You'll need to remove either this file or the other.

Reply:
nope this one is using sequential implementation while document import_datas.py is using multiprocess

Reply:
We don't need the sequential one. That's the point of the multiprocessing version. We should never have to use it again since it is slower in every aspect.